### ISSUE TITLE
allow events descriptions to contain 0 values

### ISSUE THREAD

**Issue by compmonks** [bids-standard/pybv#93]
I have an event file with 'labels' (description) which contain 0 values. 
In this case, the labels are rotations in degrees but `write_brainvision ` fails during `_chk_events` and returns the following error: `events: when `type` is Stimulus, descriptions must be positve ints.`

Would it be possible to allow `event["description"]` to be at least >= 0 instead of >=1? The current test is 
```
443 if event["description"] < 1:
444                 raise ValueError(f"events: when `type` is {event['type']}, "
445                                  "descriptions must be positve ints.")```

**Comment by compmonks** [comment]
@sappelhoff , thanks! I just created one there: 7afe170aebbdd7b23462c64866599577fb83b833.

**Comment by sappelhoff** [comment]
closed in #95
